configs/run.config

```diff
@@ -5,6 +5,8 @@
 params {
     mode = "run"
 
+
+    read_type = "paired_end"
```

Comment:
This should be unnecessary, we can figure this out based on the format of the sample sheet.

Reply:
Good idea, will implement.

Reply:
Did this ever get resolved?